<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FoodAndDrinks</class>
 <widget class="QDialog" name="FoodAndDrinks">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Food &amp; Drinks</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_10">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Popcorn</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Traditional</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>$8.79</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_traditional">
            <property name="text">
             <string>Select</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Kids Pack</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>$7.09</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_kidsPacks">
            <property name="text">
             <string>Select</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Beverages</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_17">
          <item>
           <widget class="QLabel" name="label_29">
            <property name="text">
             <string>Coke</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_30">
            <property name="text">
             <string>$6.49</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_coke">
            <property name="text">
             <string>Select</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_22">
          <item>
           <widget class="QLabel" name="label_39">
            <property name="text">
             <string>ICEE</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_40">
            <property name="text">
             <string>$6.99</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_ICEE">
            <property name="text">
             <string>Select</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_23">
          <item>
           <widget class="QLabel" name="label_41">
            <property name="text">
             <string>Bottled Drinks</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_42">
            <property name="text">
             <string>$4.99</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_bottledDrinks">
            <property name="text">
             <string>Select</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_24">
          <item>
           <widget class="QLabel" name="label_43">
            <property name="text">
             <string>Costa Coffee</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_44">
            <property name="text">
             <string>$1.00</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_costaCoffee">
            <property name="text">
             <string>Select</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Snacks</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_11">
          <item>
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>Chicken Tenders</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_18">
            <property name="text">
             <string>$8.29</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_chickenTenders">
            <property name="text">
             <string>Select</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_12">
          <item>
           <widget class="QLabel" name="label_19">
            <property name="text">
             <string>Curly Fries</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_20">
            <property name="text">
             <string>$5.99</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_curlyFries">
            <property name="text">
             <string>Select</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>Treats</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_13">
          <item>
           <widget class="QLabel" name="label_21">
            <property name="text">
             <string>Ice Cream</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_22">
            <property name="text">
             <string>$4.69</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_iceCream">
            <property name="text">
             <string>Select</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_14">
          <item>
           <widget class="QLabel" name="label_23">
            <property name="text">
             <string>Gluten Free Snacks</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_24">
            <property name="text">
             <string>$6.29</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_glutenFreeSnacks">
            <property name="text">
             <string>Select</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_15">
          <item>
           <widget class="QLabel" name="label_25">
            <property name="text">
             <string>Candy</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_26">
            <property name="text">
             <string>$4.89</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_candy">
            <property name="text">
             <string>Select</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="title">
        <string>Special Offers</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_18">
          <item>
           <widget class="QLabel" name="label_31">
            <property name="text">
             <string>Wonder Woman 1984</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_32">
            <property name="text">
             <string>$11.99</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_wonderWoman">
            <property name="text">
             <string>Select</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_6">
       <property name="title">
        <string>Party Packs</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_19">
          <item>
           <widget class="QLabel" name="label_33">
            <property name="text">
             <string>Cameo Party Pack</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_34">
            <property name="text">
             <string>$79.99</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_cameoPartyPack">
            <property name="text">
             <string>Select</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_20">
          <item>
           <widget class="QLabel" name="label_35">
            <property name="text">
             <string>Kids Party Pack</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_36">
            <property name="text">
             <string>$49.99</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_kidsPartyPack">
            <property name="text">
             <string>Select</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_21">
          <item>
           <widget class="QLabel" name="label_37">
            <property name="text">
             <string>Tenders &amp; Fries Pack</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_38">
            <property name="text">
             <string>$79.99</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_tendersFriesPack">
            <property name="text">
             <string>Select</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QTableWidget" name="tableWidget">
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>200</number>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderStretchLastSection">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Item</string>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Price</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Total:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_foodCost">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;$33.50&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_clearAll">
         <property name="text">
          <string>Clear All</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_checkOut">
         <property name="text">
          <string>Check out</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
